docs/rule/element-attributes-order.md

```diff
@@ -0,0 +1,38 @@
+## element-attributes-order
+
+Element attributes must be ordered as:
+
+* arguments
+* attributes
+* element modifiers
+* ...attributes
```

Comment:
it can be valuable to put some attributes **after** `...attributes`, but it seems that this rule doesn't allow that? 🤔

Reply:
this behavour supported https://github.com/ember-template-lint/ember-template-lint/pull/871/files#diff-780c04b6663870e4456bae29ebc13842R16